### PR TITLE
503 alternative names

### LINKED ISSUE #503: Alternative names
Alternative names have been provided in the csv file, for example Acetylsalicylic Acid == Aspirin

Image

These aren't loaded into dgraph or available to search.

Acceptance Criteria

- [ ] Alternative names are added to dgraph
- [ ] Alternative names can be added via the new admin UI
- [ ] Searching using an alternative name should find the drug in the Browse page

### PR DESCRIPTION
**Pull request by lache-melvin**
Fixes #503

## Description

Adds alternative names for products:
Screenshot 2024-01-25 at 4 04 28 PM

Add new ones - same pending change process as is currently in place. Have disabled editing existing ones to be consistent but its not really necessary, we could leave them editable?
Screenshot 2024-01-25 at 4 05 22 PM

Main search and fuzzy search now search across description, code and alt names:
Screenshot 2024-01-25 at 4 10 52 PM

Adds product synonyms column to the consumable and vaccine csvs.

### CODE REVIEW

File: backend/service/src/universal_codes/entity_filter.rs
Comment (lache-melvin) [inline backend/service/src/universal_codes/entity_filter.rs:151]: Do you think we'd want the exact match or contains for the codes here?

Comment (jmbrunskill) [reply backend/service/src/universal_codes/entity_filter.rs:151]: I think equal, because we're usually searching for a copy and pasted code.
I think potentially we could do starts with? Like if you want to type in part of the code if you aren't copy and pasting...
But I don't think we want a search for `ace` to pick up `41ace4bf`?

File: frontend/system/src/Admin/EditEntity/EditEntityView.tsx
Comment (jmbrunskill) [inline frontend/system/src/Admin/EditEntity/EditEntityView.tsx:13]: Haven't seen isFetchedAfterMount before.

**Review by jmbrunskill — APPROVED** [review]
This is awesome, works great!

**Comment by jmbrunskill** [comment]
> Add new ones - same pending change process as is currently in place. Have disabled editing existing ones to be consistent but its not really necessary, we could leave them editable?
I think making editible would be handy, I think we'll end up making things editiable eventually. But maybe have a extra permission, or a lock icon to enable editing? I'm find with how it is for this PR...

File: frontend/system/src/Admin/EditEntity/EditEntityView.tsx
Comment (lache-melvin) [reply frontend/system/src/Admin/EditEntity/EditEntityView.tsx:13]: Yeah this was a bit of a weird one, but it was loading the edit forms with stale data after the approved pending change. It was immediately going and fetching the new data, like it new the data was invalid, but the form rendered with the initial stale stuff... so this waits until the refetch is complete 🤷‍♀️

File: backend/service/src/universal_codes/entity_filter.rs
Comment (lache-melvin) [reply backend/service/src/universal_codes/entity_filter.rs:151]: Yeah nice, I'll make it starts_with

**Pull request merged**
